root/release/edit/recordings.tt

```diff
@@ -91,7 +91,7 @@
       <div class="edit-recording">
         <div class="buttons">
           <button type="button" data-bind="disable: loading" data-click="loadTracks">
-            [% l('Edit') | html_entity %]
+            [% lp('Edit', 'button') | html_entity %]
```

Comment:
Button is just the presentation, sure `button` context helps with context, but it does not help with the meaning of Edit (could still be either _an_ edit or _to_ edit). Would `action` be a better fit for example?

Reply:
The question is not whether it's a verb or a noun. The point is that some languages (such as mine) would need the verb to appear differently depending on whether it's used as in a title-like position (e.g. as a tab in the UI, as a column header, as part of page title, or as menu header), or as a direct command (whether button, link or a menu command).

FWIW, I changed the context to "button/link" later on.

I feel that "action" can be interpreted a bit too ambiguously... Would "command" instead of "button/link" make more sense?

Reply:
> The question is not whether it's a verb or a noun. [...] the verb [...]

I would not assume it is always the verb as in _Edit_ tab ([example](https://musicbrainz.org/release/0ce274e3-3b89-4d15-94ac-4f92e497ea60/edit)), it could be the noun as in _Edit_ descriptive page ([example](https://musicbrainz.org/edit/65697443)).

> to appear differently depending on whether it's used as in a title-like position (e.g. as a tab in the UI, as a column header, as part of page title, or as menu header), or as a direct command (whether button, link or a menu command). [...] I feel that "action" can be interpreted a bit too ambiguously... Would "command" instead of "button/link" make more sense?

I think I got it. We just had different expectations of the context string: meaning/disambiguation vs. role/layout. It seems that we should probably have both. That would require to agree on how to formulate context strings. Since MBS-10132 was not specific enough about that, let's discuss guidelines about writing localizable strings on that ticket?

Reply:
> I would not assume it is always the verb as in _Edit_ tab ([example](https://musicbrainz.org/release/0ce274e3-3b89-4d15-94ac-4f92e497ea60/edit)), it could be the noun as in _Edit_ descriptive page ([example](https://musicbrainz.org/edit/65697443)).

Well, you are right of course, it's just that _that_ is a third context in addition to my proposed two :) Luckily, currently none of the nouns "edit" are confused with the verbs "edit" (or at least I haven't noticed it), and it's only the verbs that I added context to.

> I think I got it. We just had different expectations of the context string: meaning/disambiguation vs. role/layout. It seems that we should probably have both.

That would probably be a better option indeed.

> That would require to agree on how to formulate context strings. ~Since [MBS-10132](https://tickets.metabrainz.org/browse/MBS-10132) was not specific enough about that~, oops [MBS-5380](https://tickets.metabrainz.org/browse/MBS-5380), let's discuss guidelines about writing localizable strings on that ticket? Or create a new ticket to improve https://musicbrainz.org/doc/Server_Internationalisation?

Sure. Maybe it would make sense to create a new ticket, so that MBS-5380 would only have the actual requests to split certain strings?

Reply:
Agreed, created MBS-10739.